practices/images-best-practices.md

```diff
@@ -0,0 +1,48 @@
+# Best practices when dealing with images for the Web
+
+In a front-end workflow we typically deal with two kinds of images:
+
+* **Icons, logos, buttons and other graphical elements** that are part of the User Interface of the page. These are usually managed by the frontend, and so they are stored with the rest of resources (templates, CSS, JS). Like those resources, they may be shared between many different pages.
+
+* **Content images**. These are unique to a page (or a handful of pages). They will be usually provided from a database or a content management system.
+
+Most best practices are common to both categories.
+
+## Formats
+
+### What format to use
+
+* Use SVGs whenever you want to display a vector image.
+* Use JPEGs for photo images, or images with many little details.
+* Use PNGs for graphs and other images with large areas of the same colour. Also for images that require transparency.
+* Don't use GIFs.
+
+### About WebP
+
+We don't encourage the use of the [WebP format](https://developers.google.com/speed/webp/) as support for it is still not widespread. [MS is not planning to add support to Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webpimageformat/) at the moment. Firefox is currently working in adding support for it, so we may reconsider after that.
```

Comment:
I don't personally aspire to use WebP, I don't think that it's a good solution, for many reasons that don't deserve to be written on a first draft, as I'm mostly interested in writing down other stuff that I have in the pipeline. I'm mostly trying to document what we're currently doing.

So I guess that this bows down to:
* Are we using WebP anywhere, and why?
* If not, are we going to use it soon, and why?

Even if the document is _partly_ aspirational, I think that saying "you should be using WebP" when we're not using it ourselves, nor planning to use it, is misleading.

But I feel that that discussion don't belong to this PR, so seeing as this is a contentious point I'm removing the GIF and WebP sections altogether so we can get this merged.